src/op_mode/openvpn.py

```diff
@@ -142,6 +144,17 @@ def _get_interface_status(mode: str, interface: str) -> dict:
 
     return data
 
+def _get_interface_state(iface):
+    rc, out = rc_cmd(f'ip --json link show dev {iface}')
+    data = json.loads(out)
+    return data[0].get('operstate', 'DOWN')
+
```

Comment:
jestabro@eunomia# set interfaces openvpn vtun2 disable
[edit]
jestabro@eunomia# commit
[edit]
jestabro@eunomia# ip --json link show dev vtun2
Device "vtun2" does not exist.